### ISSUE TITLE
amd64 does not works pre compiled?

### ISSUE THREAD

**Issue by stivyw** [adnanh/webhook#684]
```
# wget https://github.com/adnanh/webhook/releases/download/2.8.1/webhook-linux-amd64.tar.gz
saving to 'webhook-linux-amd64.tar.gz'
webhook-linux-amd64. 100% 
'webhook-linux-amd64.tar.gz' saved
# tar -xvf webhook-linux-amd64.tar.gz
webhook-linux-amd64/
webhook-linux-amd64/webhook
# mv webhook-linux-amd64/webhook /usr/local/bin/
# chmod +x /usr/local/bin/webhook 
# webhook 
sh: webhook: not found
# arch
x86_64
```

**Comment by adnanh** [comment]
Try running with `./webhook`

**Comment by pecataToshev** [comment]
I had the same problem. I was using an alpine linux 3.19 and had the same output. Rebuilding the binary myself solved the issue.

**Comment by ianroberts** [comment]
The 2.8.1 linux-amd64 binary release is dynamically linked against glibc, so does not work on Alpine (with its musl libc).  The newly-released 2.8.2 version binary release is statically linked so should be fine on Alpine.
